lib/matplotlib/testing/decorators.py

```diff
@@ -134,10 +135,15 @@ def copy_baseline(self, baseline, extension):
             # os.symlink errors if the target already exists.
             with contextlib.suppress(OSError):
                 os.remove(expected_fname)
-            try:
-                os.symlink(orig_expected_path, expected_fname)
-            except OSError:  # On Windows, symlink *may* be unavailable.
+            if ('microsoft-standard' in uname().release  # WSL2
+                    or 'Microsoft' in uname().release):  # WSL1
```

Comment:
`if 'icrosoft' in ...`?

Reply:
`'icrosoft'` will be changed to `'microsoft'` in the next few years when someone discovers a "typo" and we have all forgotten this conversation 🤣 .

 `'microsoft' in uname().release.lower()` would be my suggestion to balance short with readable.

That said we should not spend too much time on this discussion...